thorlcr/master/thmastermain.cpp

```diff
@@ -480,7 +480,13 @@ class CThorEndHandler : public CSimpleInterface, implements IThreaded
         if (stopped)
             return;
         if (!sem.wait(timeout))
-            raise(SIGKILL);
+        {
+#ifdef _WIN32
+            TerminateProcess(GetCurrentProcess(), 1);
+#else
+            _exit(1);
```

Comment:
wouldn't _exit be equally applicable to Windows?

Reply:
Yes I think so.  I copied code from somewhere else.  Let's just use _exit(1) for both.